config/initializers/saml_idp.rb

```diff
@@ -0,0 +1,60 @@
+require 'service_provider'
+
+SamlIdp.configure do |config|
+  protocol = Rails.env.development? ? 'http://' : 'https://'
+  api_base = protocol + Figaro.env.domain_name + '/api'
+
+  config.x509_certificate = Rails.application.secrets.saml_cert
+  config.secret_key = OpenSSL::PKey::RSA.new(
+    File.read(Rails.root + 'config/saml.key.enc'),
+    Figaro.env.saml_passphrase).to_pem
+
+  # config.signature_alg = 'rsa-sha256'
+  # config.digest_alg = 'sha256'
+
+  # Disabled because authnrequest signing doesn't prevent any
+  # attack that can be thought of.
+  # config.verify_authnrequest_sig = true
```

Comment:
> I don't understand how we prevent the user from being redirected to a malicious URL (along with whatever attribs we include in the SAMLResponse) in a seemingly valid authn request?

I was getting confused with another IdP that allows such redirects. @amoose reminded me that we [whitelist our SPs along with the redirect URLs](https://github.com/18F/identity-idp/blob/master/config/service_providers.yml)